src/plugins/oSnap/types.ts

```diff
@@ -405,3 +405,24 @@ export type OGProposalState =
   | (AssertionTransactionDetails & {
       status: 'transactions-executed';
     });
+
+interface ResultUrl {
+  url: string; // This is the URL to the simulation result page (public or private).
+  public: boolean; // This is false if the project is not publicly accessible.
```

Comment:
for frontend, we might also validate that this is true, otherwise the returned url will not be accessible. Normally API is expected to run with public project, but its better to check it

Reply:
Ah great thanks Reinis! I think, correct me if I'm wring, but we've configured the API to always be using UMA's project since we're always interacting with the OG. Will this ever change? I suppose it _might_ in which case we'd probably have to show some sort of warning that the link _might_ not be accessible to the user.

Reply:
yes, API uses public Tenderly project credentials, but still worth checking as this can be changed in project settings